Error adding symbols: DSO missing from command line

I just tied to update from the master-branch (git pull) and tried to build.
At the end, it throws this error:
```
[ 96%] Building CXX object CMakeFiles/libgerbera.dir/src/zmm/exception.cc.o
[ 97%] Building CXX object CMakeFiles/libgerbera.dir/src/zmm/stringtokenizer.cc.o
[ 98%] Building CXX object CMakeFiles/libgerbera.dir/src/zmm/object.cc.o
[ 99%] Building CXX object CMakeFiles/libgerbera.dir/src/zmm/strings.cc.o
[ 99%] Built target libgerbera
Scanning dependencies of target gerbera
[ 99%] Building CXX object CMakeFiles/gerbera.dir/src/main.cc.o
[100%] Linking CXX executable gerbera
/usr/bin/ld: CMakeFiles/libgerbera.dir/src/metadata/ffmpeg_handler.cc.o: undefined reference to symbol 'av_log_set_callback@@LIBAVUTIL_56'
/usr/bin/ld: //usr/lib/x86_64-linux-gnu/libavutil.so.56: error adding symbols: DSO missing from command line
collect2: error: ld returned 1 exit status
CMakeFiles/gerbera.dir/build.make:322: recipe for target 'gerbera' failed
make[2]: *** [gerbera] Error 1
CMakeFiles/Makefile2:109: recipe for target 'CMakeFiles/gerbera.dir/all' failed
make[1]: *** [CMakeFiles/gerbera.dir/all] Error 2
Makefile:129: recipe for target 'all' failed
make: *** [all] Error 2
``
A quick search via Google did not result in a solution. I found this: https://stackoverflow.com/questions/19901934/strange-linking-error-dso-missing-from-command-line

--- a/cmake/FindFFMPEG.cmake
+++ b/cmake/FindFFMPEG.cmake
@@ -85,7 +85,7 @@ get_filename_component(FFMPEG_INCLUDE_DIR ${FFMPEG_INCLUDE_DIR} ABSOLUTE)
 CHECK_STRUCT_HAS_MEMBER("struct AVStream" codecpar libavformat/avformat.h HAVE_AVSTREAM_CODECPAR LANGUAGE C)
 
 FIND_PACKAGE(PkgConfig QUIET)
-PKG_CHECK_MODULES(FFMPEG QUIET libavformat)
+PKG_CHECK_MODULES(FFMPEG QUIET libavformat libavutil)
 IF (NOT FFMPEG_FOUND)
 	FIND_LIBRARY(FFMPEG_avformat_LIBRARY avformat
 		/usr/local/lib

